<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankDialog</class>
 <widget class="QDialog" name="RankDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>540</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>741</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 50pt &quot;楷体&quot;;
color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>排行榜</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
